practice-app/event_app/views.py

```diff
@@ -91,3 +91,26 @@ def viewGithubInfo(req):
     except Exception as e:
         print(str(e))
         return HttpResponseRedirect('../boun/viewGithubInfoPage?fail=true')
+
```

Comment:
You should create another function named addEventPage to redirect user to Event Form.